query.py

```diff
@@ -165,10 +165,19 @@ def extractFormatsMap(report):
 
 def format_supported_with_optimal_tiling_features(formats_map, format, flags):
     return format in formats_map and (formats_map[format]['optimalTilingFeatures'] & flags) == flags
+
+
 def format_supported_with_linear_tiling_features(formats_map, format, flags):
     return format in formats_map and (formats_map[format]['linearTilingFeatures'] & flags) == flags
 
 
+def try_to_int(value):
+    if type(value) == str:
```

Comment:
nit: I changed this from a try-except to a type check (double checked it still works)